modules/administration/pages/recurring-actions.adoc

```diff
@@ -0,0 +1,76 @@
+[[recurring-actions]]
+= Recurring Actions
+
+Scheduling recurring actions allows you to manage schedules for automated recurring highstate execution on minion, group and organization level depending on the frequency you choose.
```

Comment:
```suggestion
Scheduling recurring actions allows you to manage schedules for automated recurring highstate execution on minion, group, and organization level depending on the frequency you choose.
```
And "client" or "Salt client" instead of "minion"? If yes, we should also change it later on.

Reply:
Just "Salt clients" in our case not traditional ones.